### DISCUSSION TITLE
Where is the built-in CSS file so that I can modify just two things when exporting to PDF

### DISCUSSION THREAD

**Discussion by mousetrappling** [Zettlr/Zettlr#3633] [Q&A] [answered]
Hi all!

I'm trying to very slightly tweak the output to PDF – make the links blue and the text justified – but I can't figure out how to do that.

To be clear, I'm talking about the export option that is called PDF Document **not** the one that's called PDF (LaTeX).  By default that's a lot closer to what I want the output to look like, if I use the PDF (LaTeX) option then I need to do a lot more to the style.

I can see a place in the defaults file to add a CSS file, and I figured out I can also do that in the YAML header of my file.  But the problem is that then I need to know all the other things that the built-in CSS file sets up.

I looked around on my file system to try and find the built-in CSS so I can copy it and tweak the two things I want to tweak but have had no success finding it.

I've tried to look in the discussions here and search the documentation and online, but either I find things about styling the editor or information about exports pre-2.0 which is no longer relevant.

Is there a way to override individual things in my YAML frontmatter that I've missed?  Otherwise, I'd like to know where the built-in CSS comes from so I can copy and modify it.

Thanks,
MP.

**Comment by nathanlesage** [comment, marked as answer]
By default, the exporter will export your files to HTML and then "print" it to PDF (like in your browser). Therefore it's using the default HTML template from Pandoc (unless you specified something else in your defaults).

You can find the default HTML template in the Pandoc repo (https://github.com/jgm/pandoc) under data -> templates. This should show what you need to tweak.

**Reply by mousetrappling** [reply]
Thank you!
